de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/fskdbview/FSKDBViewNodeFactory.xml

```diff
@@ -9,15 +9,42 @@
 
 	<fullDescription>
 		<intro>
-
+            This node visualizes FSK models as a HTML Table.
+            The source of this metadata can be read from remote repository which can be set via the "Repository URL" option in the setting window.
+            Also this metadata can be provided as KNIME table via the input port.<br/>
+            <br/>
+            <a href="https://kni.me/w/hvgtZc8MYdY7scqc">Example Workflow</a>
+            <br/>
+            <br/>
 		</intro>
+		<option name="Repository location">
+            By default refers to <a href="https://knime.bfr.berlin/landingpage">https://knime.bfr.berlin/landingpage</a>
+        </option>
```

Comment:
@ahmadswaid I should point out, and that has been a little weird in the past, that when using this url as described above: https://knime.bfr.berlin/landingpage , the db view doesn't work. You have to add the / at the end of the URL: 
https://knime.bfr.berlin/landingpage/ .